### PR TITLE
Add Kupo to `test-nixos-configurations.nix`

### PR DESCRIPTION
**Pull request by Bradley-Heather**
Closes # .

### Pre-review checklist

- [ ] All code has been formatted using our config (`make format`)
- [ ] Any new API features or modification of existing behavior is covered as defined in our [test plan](https://github.com/Plutonomicon/cardano-transaction-lib/blob/develop/doc/test-plan.md)
- [ ] The changelog has been updated under the `## Unreleased` header, using the appropriate sub-headings (`### Added`, `### Removed`, `### Fixed`), and the links to the appropriate issues/PRs have been included

### CODE REVIEW

**Review by jy14898 — DISMISSED** [review]

File: nix/test-nixos-configuration.nix [hunk 1]
```diff
@@ -38,6 +38,15 @@
     nodeSocket = "/var/run/cardano-node/node.socket";
   };
 
+  services.kupo = {
```
[the remark this answers is not in the document]
Comment (Bradley-Heather) [reply nix/test-nixos-configuration.nix:42]: Thanks. done

File: flake.nix [hunk 2]
```diff
@@ -48,6 +48,7 @@
     { self
     , nixpkgs
     , cardano-configurations
+    , kupo
```
[the remark this answers is not in the document]
Comment (Bradley-Heather) [reply flake.nix]: yup

File: nix/test-nixos-configuration.nix [hunk 3]
```diff
@@ -10,6 +10,7 @@
       { from = "host"; host.port = 1337; guest.port = 1337; }
       { from = "host"; host.port = 8081; guest.port = 8081; }
       { from = "host"; host.port = 9999; guest.port = 9999; }
+      { from = "host"; host.port = 1442; guest.port = 1442; }
```
Comment (jy14898) [inline nix/test-nixos-configuration.nix:13]: Can you merge develop and the comment for kupo here?

**Review by jy14898 — DISMISSED** [review]

File: nix/test-nixos-configuration.nix (hunk 3 above)
Comment (klntsky) [reply nix/test-nixos-configuration.nix:13]: Bradley is on vacation, I'm taking over

**Review by klntsky — APPROVED** [review]

**Pull request merged**